modules/platforms/dotnet/Apache.Ignite/Internal/Proto/MessagePackReaderExtensions.cs

```diff
@@ -137,8 +137,8 @@ public static unsafe IgniteUuid ReadIgniteUuid(this ref MessagePackReader reader
             var size = ValidateExtensionType(ref reader, ClientMessagePackType.IgniteUuid);
             var bytes = reader.ReadRaw(size);
 
-            var res = default(IgniteUuid);
-            bytes.CopyTo(new Span<byte>(res.Bytes, size));
+            IgniteUuid res = default;
+            bytes.CopyTo(new Span<byte>(&res, size));
             res.Size = (byte)size;
```

Comment:
Should we add an assertion here, that size is less than 127?

Reply:
We know that `IgniteUuid` is no more than 24 bytes. Added an assertion anyway.